logstash_formatter/__init__.py

```diff
@@ -84,32 +84,28 @@ def format(self, record):
         except (KeyError, IndexError):
             pass
 
-        if 'msg' in fields:
-            fields.pop('msg')
-
-        if 'exc_info' in fields:
-            if fields['exc_info']:
-                formatted = tb.format_exception(*fields['exc_info'])
-                fields['exception'] = formatted
-            fields.pop('exc_info')
+        if fields.get('exc_info', None):
```

Comment:
thats right I was thinking of dict.pop where if you don't supply the default it throws an exception.